src/commons/application/reducers/__tests__/SessionReducer.ts

```diff
@@ -153,7 +155,10 @@ test('SET_ASSESSMENT_CONFIGURATIONS works correctly', () => {
       buildSolution: false,
       isContest: false,
       hoursBeforeEarlyXpDecay: 48,
-      earlySubmissionXp: 200
+      earlySubmissionXp: 200,
+      isManuallyGraded: false,
+      displayInDashboard: true,
+      hasTokenCounter: false
```

Comment:
will change this on my part after out merge to include the "is auto published" flag